jdaviz/core/helpers.py

```diff
@@ -166,20 +165,30 @@ def viewers(self):
                 for viewer in self.app._viewer_store.values()}
 
     @property
+    @deprecated(since="4.2", alternative="plugins['Model Fitting'].fitted_models")
     def fitted_models(self):
         """
+        Deprecated and will be removed in jdaviz 4.4.
```

Comment:
You can use a proper `.. deprecated::` directive for better rendered output: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated

Reply:
But... I think the decorator already does that for you, so no need to repeat in docstring: https://github.com/astropy/astropy/blob/fe715540195e94d6836bc639c9a312aceaf99849/astropy/utils/decorators.py#L150

Similar comment applies to all occurrences.